src/lib/libidbfs.js

```diff
@@ -85,10 +85,14 @@ addToLibrary({
             if (n.memfs_stream_ops.close) return n.memfs_stream_ops.close(stream);
           };
 
+          // Persist IndexedDB right away if the node we just created is a directory
```

Comment:
Can you explain why this is needed?  (Either here or in the PR description)

Also, is there some kind of test we could write for this?

Reply:
I see that makes sense.

How about updating the comment/description to say something like.  "Force a persist now since there is not corresponding `close` call for newly creating directories".

Reply:
Does the same problem exist for folks that directly call `mknod` to make files?  (i.e. they never get closed, so the persist is never triggered?)